### ISSUE TITLE
[BUG] - [v 0.14.0] - Icon and local storage get 400 error

### ISSUE THREAD

**Issue by pierrejo** [langgenius/dify#11763]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Dify version

0.14.0

### Cloud or Self Hosted

Self Hosted (Docker)

### Steps to reproduce

Update to 0.14.0

### ✔️ Expected Behavior

Just show Logo / images (yes, i try to re-upload, but same error)

### ❌ Actual Behavior

don't show Logo / Images and get `400`

```bash
/_next/image?url=%2Ffiles%2F787cf5f9-364c-437b-94cc-e928e4f2765f%2Ffile-preview%3Ftimestamp%3D1734454781%26nonce%3Daf57291fa7c3f47c20ba8868fc26947f%26sign%3DJDwz4HFfz_1Dvf50ZUEmZNzJMwxKjbN54K-b7rZdvN0%3D&w=2048&q=75:1 
        
        
       Failed to load resource: the server responded with a status of 400 ()
```

```bash
The requested resource isn't a valid image.
```

![2024-12-17_18-00](https://github.com/user-attachments/assets/d3ef53c7-9d8c-4550-8de8-678151b2a54b)
![2024-12-17_18-06](https://github.com/user-attachments/assets/62b52fae-ac5c-421a-94a9-a3dc925f5f4c)

**Comment by crazywoola** [comment]
Already fixed in #11701

**Comment by MaxHouxu** [comment]
1.0.0-beta.1 this bug appear again
